modules/PROGENY.py

```diff
@@ -0,0 +1,288 @@
+from settings import Config
+from common.HGNCParser import GeneParser
+import sys
+import logging
+import datetime
+import opentargets.model.core as opentargets
+import opentargets.model.bioentity as bioentity
+import opentargets.model.evidence.core as evidence_core
```

Comment:
i'd recommend using the `pjs` package https://github.com/cwacek/python-jsonschema-objects